docker/.env.example

```diff
@@ -78,3 +78,10 @@ SUPABASE_PUBLIC_URL=http://localhost:8000
 
 # Enable webp support
 IMGPROXY_ENABLE_WEBP_DETECTION=true
+
+# Configure Logflare API Calls
+LOGFLARE_API_KEY=api-key
+LOGFLARE_URL=http://localhost:54328
```

Comment:
Are we going to update the self-hosted compose file in a separate PR to use this port?

Reply:
Yes it will be handled here https://github.com/supabase/supabase/pull/12583